assets/data-port/export/export-select-content-page.js

```diff
@@ -0,0 +1,52 @@
+import { Button, CheckboxControl } from '@wordpress/components';
+import { useReducer } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { postTypeLabels } from '../../shared/helpers/labels';
+
+export const ExportSelectContentPage = ( { onSubmit } ) => {
+	const [ values, setValue ] = useReducer(
```

Comment:
As we are using this without the actions/reducer patterns, WDYT to change for a `useState`?

Reply:
Yes, this is a simple way that just merges the payload into the state, which is a pretty common case. I made it into an explicit helper in the next PR: https://github.com/Automattic/sensei/pull/3413/commits/17ff8d3cc81b0bab4606997b85527d6024123111#diff-d0b8bbdad55e41ec8974ba1418c55497